compute-budget/src/compute_budget.rs

```diff
@@ -241,12 +242,13 @@ impl ComputeBudget {
     pub fn get_compute_budget_and_limits(
         &self,
         compute_budget_limits: &ComputeBudgetLimits,
+        transaction_fee: u64,
```

Comment:
if we pass fee details here, we could change definition (and name) of `FeeBudgetDetails`, since we really only need the budget details then.

Reply:
I didn't find `FeeBudgetDetails` in repo. Did you mean `FeeBudgetLimits` or `FeeDetails`?